build.gradle

```diff
@@ -627,8 +630,8 @@ try {
             defineProperty("jdkBuildNumber", jdkVersionInfo[1])
 
             // Define global properties based on the version of Java
-            def status = compareJdkVersion(jdkVersion, "12")
-            ext.jdk12OrLater = (status >= 0)
+//            def status = compareJdkVersion(jdkVersion, "18")
+//            ext.jdk18OrLater = (status >= 0)
```

Comment:
Why do we have this if it is commented out ?

Reply:
It's meant to be an example of how to do a conditional based on the JDK version. I could have just removed it, but I'll add a comment to make that clear.